morphs/data/behavior.py

```diff
@@ -0,0 +1,42 @@
+'''Functions for loading and processing behavior data'''
+import morphs
+import pandas as pd
+from google_drive_downloader import GoogleDriveDownloader as gdd
+
+
+def reduce_behave_data(df):
+    '''Trims df and changes dtypes to reduce pkl size'''
+    df = df[['class_', 'response', 'correct', 'rt', 'reward', 'stim_id', 'subj']]
+    for col in ['correct', 'reward']:
+        df[col] = df[col].astype(bool)
+    for col in ['class_', 'response', 'subj']:
+        df[col] = df[col].astype('category')
+    return df
+
+
+def gen_behavior_df():
+    '''Loads raw data and generates pkl containing all important behavior data'''
+    import behav
+    from behav import loading
+    behav_data = loading.load_data_pandas(morphs.subj.BEHAVE_SUBJS,
```

Comment:
This looks a little bit like a magic number. Not entirely sure of a good pattern, maybe passing the subjects in to the function, or are you trying to isolate the subjects as part of the config?

Reply:
I think this may just be a reflection of me not being able to distinguish your configuration files from your actual python files. For example, morphs.utils.logistic and morphs.utils.paths both look like python, but paths is really for config, as is subj, whereas logistic is an actual implementation.